src/lib/components/ui/ArticleSpotlight.svelte

```
@@ -2,26 +2,30 @@
 	import type { ArticleMetadata } from '$lib/types/article';
 	import Badge from './badge/badge.svelte';
 	import { goto } from '$app/navigation';
+	import { getAuthorDisplayName, getAuthorsText } from '$lib/utils/authors';
```

ESLint

[error] 'getAuthorDisplayName' is defined but never used.

(@typescript-eslint/no-unused-vars)